### PR TITLE
Use SendNoticeSync so pending notices are flushed with Flush.

### PR DESCRIPTION
**Pull request by vmihailenco**
Uses new API added in https://github.com/airbrake/gobrake/pull/14

### CODE REVIEW

File: airbrake.go [hunk 1]
```diff
@@ -11,6 +12,8 @@ import (
 // logs to Airbrake.
 var Gobrake *gobrake.Notifier
 
+var gobrakeWG sync.WaitGroup
```
Comment (zefer) [inline airbrake.go]: Is this being used anywhere?

Comment (vmihailenco) [inline airbrake.go]: Nope, removing.

**Comment by vmihailenco** [comment]
PTAL

File: airbrake.go [hunk 2]
```diff
@@ -55,12 +55,12 @@ func notifyAirbrake(s severity, format string, args ...interface{}) {
 			continue
 		}
 
-		notice := Gobrake.Notice(err, req, 4)
+		notice := Gobrake.Notice(err, req, 2)
```
Comment (zefer) [inline airbrake.go]: Why did the stack depth need to change, here (from 4 to 2)?

Comment (vmihailenco) [inline airbrake.go]: I can't say for sure, but this appears to be the new right value.

**Comment by zefer** [comment]
LGTM

**Pull request merged**